### ISSUE TITLE
Re-initialize ControlIDs

### ISSUE THREAD

**Issue by waldo1001** [NAVDEMO/RenumberNavObjectIds#2]
You are able to analyze ControlIDs from Delta's.

Why not add the ability to re-initialize the control-id's in a delta?  Basically the same as what you have, but not having a "fromObjectId" ... just all controlids, starting from a number.

This would be useful in a lot of cases - not only renumbering.

**Comment by freddydk** [comment]
The problem is, that I am using text search.
I am avoiding stuff that I locate in strings, dotnet declarations and more - but if you have a line in your code saying: Price := 52000 - then that will be renumbered as well.
Normally we are working with object ranges which are not normal numbers in code - but... - if I were to search for 1, 2 and 3 - this picture changes dramatically, so I cannot do this in the renumber function.
But then again - maybe create a special function which goes through all pages and renumber all controls from an offset - that might be possible.

**Comment by waldo1001** [comment]
Yeah, you're right.  Didn't read the code enough to realize it was txt only.  I guess a manual compare after the renumber is very recommended ;-)

It should be a special function, identifying controls (which is do-able), in a delta, which would mean it's only "handling" controls that "I" added.  Well, it will probably also contain the controls that I changed ... which means I should only focus on the "insertions" in stead of the "modifications" ...

Come to think of it - it might be more complex than I thought :-/.
